frontends/ol-components/src/components/Button/Button.tsx

```diff
@@ -94,6 +94,7 @@ const ButtonStyled = styled.button<ButtonStyleProps>((props) => {
       ":disabled": {
         cursor: "default",
       },
+      minWidth: "100px",
```

Comment:
Our tab component is based on `Button` (it uses tertiary buttons). So this increases the width, of tabs, too.

@steven-hatch Do we want tabs to have min-width 100px? I can see pros/cons

<img width="1250" alt="Screenshot 2024-06-27 at 4 24 31 PM" src="https://github.com/mitodl/mit-open/assets/9010790/9a9916d3-b26c-4591-8bc8-92d5863690d8">

<img width="1632" alt="Screenshot 2024-06-27 at 4 24 42 PM" src="https://github.com/mitodl/mit-open/assets/9010790/fe190c42-c88c-45dd-a925-1d53563f706b">
<img width="472" alt="Screenshot 2024-06-27 at 4 25 01 PM" src="https://github.com/mitodl/mit-open/assets/9010790/9a15843f-969a-4fb8-aa27-97c7d029910c">

<img width="504" alt="Screenshot 2024-06-27 at 4 25 18 PM" src="https://github.com/mitodl/mit-open/assets/9010790/6ee43ea6-b36b-44f3-95c2-f92999fa405d">

Really the only con is width on mobile screens.

## Separate issue

In the screenshots above, you can see the mobile menu button in top-left corner now has wrong width.

@shanbady I'm sorry this came up... `Button` should not have been used for that in the first place.

But, could you fix it so we don't regress? Probably that menu button should be a "Snowflake". I would just apply the relevant styles to it. See figma: https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=3852-51162&m=dev

The code is in `frontends/mit-open/src/page-components/Header/MenuButton.tsx`

Reply:
Should be addressed now @ChristopherChudzicki